packages/blade/src/components/Dropdown/_decisions/decisions.md

```diff
@@ -0,0 +1,702 @@
+      <ActionListItem title="Item Value" value="item-two" />
+    </ActionList>
+  </DropdownOverlay>
+</Dropdown>
+```
+
+<br/>
+
+## Navigation
+
+- [Dropdown](#dropdown)
+  - [SelectInput](#selectinput)
+  - [DropdownOverlay](#dropdownoverlay)
+- [ActionList](#actionlist)
+  - [ActionList Parent Component](#actionlist-parent-component)
+  - [ActionListItem](#actionlistitem)
+  - [ActionListSection](#actionlistsection)
+  - [ActionListHeader](#actionlistheader)
+  - [ActionListFooter](#actionlistfooter)
+- [Why `x` instead of `y`](#why-x-instead-of-y)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+- [Referrences](#referrences)
+
+---
+
+<br/>
+
+## Dropdown
+
+`Dropdown` will be a wrapper component which will handle the logic of opening and closing of the dropdown. It won't separately render anything in UI apart from the first children. (E.g. `SelectInput` in above example)
+
+### API
+
+Usage
+
+```jsx
+<Dropdown>{/* Dropdown sub-components */}</Dropdown>
+```
+
+Props
+
+_No Props Needed_
+
+### SelectInput
+
+<img src="2022-11-24-16-33-30.png" alt="Select Field Figma Designs" width="500" />
+
+Select is an Input Field that can open a dropdown on click. It can be extended from `BaseInput` component which covers most decisions like validations, focus, etc.
+
+#### Designs
+
+- [Figma - Select](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=13590%3A171090&t=KTWGvEUBEuUdQh9K-0)
+
+#### API
+
+Sample Usage
+
+```jsx
+<SelectInput icon={CalendarIcon} label="Your favorite Design System" helpText="Select only one" />
+```
+
+Props
+
+Mostly similar to other `Input` components extended from `BaseInput`
+
+```ts
+type SelectInputProps = {
+  // ... FormLabelProps like `label`, `labelPosition`
+  // ... FormValidationProps like `validationState`, `helpText`, `errorText`, etc.
+  // ... State Props like `isRequired`, `isDisabled`,
+  icon: IconComponent;
+};
+```
+
+#### A11y
+
+Would be ideal if we can render it as `button` with `aria-expanded` prop to handle state of dropdown open and close.
+
+Reference: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded#buttons
+
+### `DropdownOverlay`
+
+An Overlay that positions dropdown correctly.
+
+Should handle the animations for dropdown.
+
+#### API
+
+Usage
+
+```jsx
+<Dropdown>
+  <SelectInput label="Click For Greetings" />
+  <DropdownOverlay>Hi</DropdownOverlay>
+</Dropdown>
+```
+
+Props
+
+_No Props Needed_
+
+<br/><br/>
+
+## ActionList
+
+Plain list that can be rendered inside dropdown, in navigation menu, or anywhere. It has selectable items that perform some action on click. It can also have multiple selectable items that render as checkboxes
+
+![](2022-12-08-16-58-46.png)
+
+### Design
+
+- [Figma - ActionList]()
+
+### API
+
+#### Simplest Usage Possible
+
+```jsx
+<ActionList
+  surfaceLevel={2}
+  onChange={(e) => {
+    console.log({ name: e.name, values: e.values });
```

Comment:
So the reason I kept it as `values` is because `ActionList` has multiple select variant, in that case it becomes closer to `CheckboxGroup` 

In `CheckboxGroup` we use `values` because we get array of all selected values.

I didn't really want to change it from `values` to `value` for single variant because it can break the UI if someone just changes the variant.